<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_dialog</class>
 <widget class="QDialog" name="main_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_dialog</string>
  </property>
  <widget class="QListWidget" name="nic_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nic_reset_btn">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>100</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="nic_info_browser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>256</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sniff_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>sniff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spoof_start_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> spoof start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spoof_stop_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>spoof stop</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="gateway_edit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
